Marketplace/Overview.md

```diff
@@ -0,0 +1,34 @@
+Visual Studio extension with many features and templates frequently used by�[Lombiq](https://lombiq.com/)�developers. Contains�[Orchard CMS](https://www.orchardcore.net/)-related (including Orchard Core and Orchard 1.x) as well as generic goodies. For Orchard developers and for other .NET developers alike!
```

Comment:
Seems like we've got a couple of gremlins (there's more below).

Reply:
WTF :(.